### PR TITLE
Merge thread count is per shard. Closes #28518

### LINKED ISSUE #28518: Elastic search documentation about merge scheduling does not explicitly state that `index.merge.scheduler.max_thread_count` is a shard level property
Elastic search documentation about [merge scheduling](https://www.elastic.co/guide/en/elasticsearch/reference/current/index-modules-merge.html) does not explicitly state that `index.merge.scheduler.max_thread_count` is a shard level property.
The documentation states that this is `The maximum number of threads that may be merging at once` but this is true only on a shard level.
Specifically, this recommendation: `If your index is on spinning platter drives instead, decrease this to 1` leads one to assume that the node disk write concurrency will be 1 which is the case only with one shard and one index on the node.

### PR DESCRIPTION
**Pull request by Sue-Gallagher**
[DOCS] Added "on a single shard" to description of index.merge.scheduler.max_thread_count. Closes 28518

### CODE REVIEW

File: docs/reference/indices/aliases.asciidoc [hunk 1]
```diff
@@ -3,14 +3,23 @@
 
 APIs in Elasticsearch accept an index name when working against a
 specific index, and several indices when applicable. The index aliases
+<<<<<<< HEAD
```
Comment (bleskes) [inline docs/reference/indices/aliases.asciidoc]: something went wrong here.

File: docs/reference/mapping.asciidoc [hunk 2]
```diff
@@ -20,8 +20,6 @@ are stored and indexed.  For instance, use mappings to define:
 Each index has one _mapping type_ which determines how the document will be
 indexed.
 
-deprecated[6.0.0,See <<removal-of-types>>].
```
Comment (bleskes) [inline docs/reference/mapping.asciidoc]: unrelevant change?

File: docs/reference/query-dsl/mlt-query.asciidoc [hunk 3]
```diff
@@ -228,8 +228,13 @@ explicitly ignore them, as for the purposes of document similarity it seems
 reasonable to assume that "a stop word is never interesting".
 
 `analyzer`::
+<<<<<<< HEAD
```
Comment (bleskes) [inline docs/reference/query-dsl/mlt-query.asciidoc]: same

**Review by bleskes — DISMISSED** [review]
Thanks @Sue-Gallagher . I think something went wrong with git and there are unintended changes in the pr.

File: docs/reference/indices/aliases.asciidoc (hunk 1 above)
Comment (Sue-Gallagher) [reply docs/reference/indices/aliases.asciidoc]: Yes, I know, and I'm sorry. As a result of being a newbie to both Elastic and Git, I've created a monster that's bent on haunting me forever. Deb has been guiding me on a twisted path toward a solution, and we hope to have it tamed in the next day or two (fingers crossed).

Comment (bleskes) [reply docs/reference/indices/aliases.asciidoc]: hehe... no worries. When you have tamed the git monster,  please teach me too :)

**Comment by debadair** [comment]
LGTM

**Pull request merged**